### ISSUE TITLE
Compatibilidad Zemax

### ISSUE THREAD

**Issue by ramezquitao** [ramezquitao/pyoptools#17]
Sería conveniente evaluar la posibilidad de crear un layer de compatibilidad con Zemax. En https://code.google.com/p/python-microscopy/source/browse/PYME/misc/zemax.py hay algo de código que podría ser útil.

**Comment by ramezquitao** [comment]
Acá hay mas código donde leen librerías de  zemax, oslo, etc  https://github.com/jordens/rayopt

**Comment by ramezquitao** [comment]
En http://www.owlnet.rice.edu/~tt3/Algorithms/read_zemax.py hay mas código para leer librerías de zemax (en este caso de materiales).

Ya hay un código base que permite importar varios elementos del catalogo de Thorlabs: Lentes, Dobletes cementados, y dobletes con gap de aire intermedio. He corrido unas pruebas simples, y la cosa parece estar funcionando.  También adicioné un catalogo de vidrios de Schott de este año, pues algunos de los lentes de Thorlabs usan este catalogo.

**Comment by ankitmaurya001** [comment]
Hi, 
Is the support for zemax files added ?

**Comment by ramezquitao** [comment]
Not really. There are some functions found in other projects that has been used to import some Zemax libraries from Thorlabs, and Edmund. But at the moment there is nothing implemented that allows to import a Zemax project to pyoptools. In principle it is doable for simple designs, using the functions we already have, but I think a full support will not be implemented in the near future because we  lack of information about Zemax files.

If you like I can point you to the code I'm talking about.

**Comment by ankitmaurya001** [comment]
Sorry for late reply as I was traveling. If you can point out the code that would be great.
I have Zemax license with me and trying to simulate some of the designs from there in pyoptools.
Does having a Zemax help in getting the missing information? Or do you suggest some other python package for this task?

**Comment by ederag** [comment]
[rayopt](https://pypi.org/project/rayopt/) seems to have a Zemax import: [zemax.py](https://github.com/quartiq/rayopt/blob/318586a0103805a49cbf177d434f6bddf66122d3/rayopt/zemax.py).

**Comment by fractures21** [comment]
Hello, can you provide the code for this website?https://www.owlnet.rice.edu/~tt3/. I can't open it now

**Comment by fractures21** [comment]
Hello, can you provide the code for this website? I can't open it now

**Comment by fractures21** [comment]
Hello, can you provide the code for this website? I can't open it now.https://www.owlnet.rice.edu/~tt3/
